tests/installation/accept_license.pm

```diff
@@ -39,10 +39,8 @@ sub run {
     if (match_has_tag('inst-welcome-no-product-list')) {
         return send_key $cmd{next} unless match_has_tag('license-agreement');
     }
-    if (check_var('INSTALLER_EXTENDED_TEST', '1')) {
-        $self->verify_license_has_to_be_accepted;
-        $self->verify_translation;
-    }
+    $self->verify_license_has_to_be_accepted;
+    $self->verify_translation if (check_var('INSTALLER_EXTENDED_TEST', '1'));
```

Comment:
We should not check for equality with the boolean true so could you just use `get_var('INSTALLER_EXTENDED_TEST')` here?